src/Microsoft.Azure.WebJobs.Extension.Storage/Blobs/Listeners/StorageBlobClientExtensions.cs

```diff
@@ -5,14 +5,13 @@
 using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Azure.WebJobs.Host.Storage.Blob;
```

Comment:
Rename file to CloudBlobClientExtensions

Reply:
Done